Add a "strict" option to the m* actions to return an error if some action fails (#2007)
kuzzleio/kuzzle: doc/2/api/controllers/document/m-update/index.md

- `refresh`: if set to `wait_for`, Kuzzle will not respond until the updates are indexed
- `retryOnConflict`: conflicts may occur if the same document gets updated multiple times within a short timespan in a database cluster. You can set the `retryOnConflict` optional argument (with a retry count), to tell Kuzzle to retry the failing updates the specified amount of times before rejecting the request with an error.
- `silent`: if set, then Kuzzle will not generate notifications <SinceBadge version="2.9.2" />

> - `strict`: if set, an error will occur if at least one document has not been updated <SinceBadge version="auto-version" />

Reviewer: ```suggestion
- `strict`: if set, an error will occur if a document was not updated <SinceBadge version="auto-version" />
```
